pallets/vesting/src/lib.rs

```diff
@@ -24,7 +24,7 @@ mod mock;
 mod tests;
 pub mod weights;
 
-use crate::weights::WeightInfo;
+pub use crate::weights::WeightInfo;
 use core::convert::TryFrom;
```

Comment:
Remove this line of code, there's no need to import `TryFrom/TryInto` in rust 2021.